docs/sources/oncall-api-reference/direct_paging.md

```diff
@@ -0,0 +1,132 @@
+---
+canonical: https://grafana.com/docs/oncall/latest/oncall-api-reference/direct_paging/
+title: Direct paging HTTP API
+weight: 1200
+refs:
+  users:
+    - pattern: /docs/oncall/
+      destination: /docs/oncall/<ONCALL_VERSION>/oncall-api-reference/users
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/alerting-and-irm/oncall/oncall-api-reference/users
+  teams:
+    - pattern: /docs/oncall/
+      destination: /docs/oncall/<ONCALL_VERSION>/oncall-api-reference/teams
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/alerting-and-irm/oncall/oncall-api-reference/teams
+---
+
+# Direct paging HTTP API
+
+## Directly page a set of users
+
+For more details about how to fetch a user's Grafana OnCall ID, refer to the [Users](ref:users) public API documentation.
+
+```shell
+curl "{{API_URL}}/api/v1/direct_paging/" \
+  --request POST \
+  --header "Authorization: meowmeowmeow" \
+  --header "Content-Type: application/json" \
+  --data '{
+    "title": "We are seeing a network outage in the datacenter",
+    "message": "I need help investigating, can you join the investigation?",
+    "source_url": "https://github.com/myorg/myrepo/issues/123",
+    "users": [
+      {
+        "id": "U281SN24AVVJX",
+        "important": false
+      },
+      {
+        "id": "U5AKCVNDEDUE7",
+        "important": true
+      }
+    ]
+  }'
+```
+
+The above command returns JSON structured in the following way:
+
+```json
+{
+  "alert_group_id": "IZHCC4GTNPZ93"
+}
+```
+
+## Directly page a team
+
+For more details about how to fetch a team's Grafana OnCall ID, refer to the [Teams](ref:teams) public API documentation.
```

Comment:
It seems we are missing a Teams API doc page...?

Reply:
Ahh right.. we have https://github.com/grafana/oncall/issues/2448

Let me add that as part of this PR and close out that issue as well

Reply:
addressed in https://github.com/grafana/oncall/pull/4815/commits/b13791e2925965a0e8b079fb95172a844fdf1dbc